README.md

```diff
@@ -7,1337 +7,97 @@
-create keys and databases.
+## Quick start
 
-First lets configure our connection to a Fauna account. If you don't have an
-account, you can create a free one [here](https://dashboard.fauna.com).
+To get started:
 
-To log in, run the following command:
+1. Install the CLI:
 
-```sh
-$ fauna cloud-login
-```
+   ```shell
+   npm install -g fauna-shell@beta
+   ```
 
-You will be prompted for your `email` and `password` from your
-[Fauna](https://dashboard.fauna.com) account.
+2. If you're using bash or zsh, enable auto-complete by appending the output of
+   `fauna completion` to your `.bashrc`, `.bash_profile`, `.zshrc,` or
+   `.zprofile`. For example:
 
-Now that we have an endpoint to connect to we can try to create a database to
-start interacting with Fauna.
+   ```shell
+   fauna completion >> ~/.zshrc
+   ```
 
-This is how you can create a database called `my_app`:
+3. Authenticate with Fauna:
 
-```sh
-$ fauna create-database my_app
-creating database my_app
+   ```shell
+   fauna login
+   ```
 
-created database my_app
+4. Run CLI commands. Specify a `--database` path, including the [Region Group
+   identifier](https://docs.fauna.com/fauna/current/manage/region-groups/#id) and
+   hierarchy, to run the command in. [Shorthand Region Group
+   identifiers](https://docs.fauna.com/fauna/current/manage/region-groups/#id)
+   are supported. For example:
 
-To start a shell with your new database, run:
+   ```shell
+   # Runs a query in the top-level 'example' database
+   # in the 'us' Region Group. Use the default admin role.
+   fauna query "Collection.all()" \
+     --database us/example
+   ```
 
-	fauna shell my_app
+## Installation
 
-Or, to create an application key for your database, run:
-
-	fauna create-key my_app
-```
-
-Now, you can start a shell within that database.
+During the beta, you can install v4 of the Fauna CLI globally using npm:
 
 ```sh
-$ fauna shell my_app
-my_app> Collection.create({ name: "Users" })
-{
-  name: "Users",
-  coll: Collection,
-  ts: Time("2023-10-03T02:40:37.060Z"),
-  indexes: {},
-  constraints: []
-}
-my_app>
-```
-
-You can also list your databases:
-
-```sh-session
-$ fauna list-databases
-listing databases
-my_app
-my_second_app
-my_other_app
+npm install -g fauna-shell@beta
 ```
 
-You can delete a particular database:
+To upgrade the CLI, run:
 
-```sh-session
-$ fauna delete-database my_other_app
-deleting database 'my_other_app'
-database 'my_other_app' deleted
 ```
-
-And you can create, list, and delete keys.
-
-This is how you create a key for the database `my_app`:
-
-```sh
-$ fauna create-key my_app
-creating key for database 'my_app' with role 'admin'
-
-created key for database 'my_app' with role 'admin'.
-secret: fnAFPULk2WAAQY9t4x0tduzuz85gC-suDbTnl7um # this will be different
-
-To access 'my_app' with this key, create a client using
-the driver library for your language of choice using
-the above secret.
+npm update -g fauna-shell@beta
```

Comment:
upgrading beta releases via npm is a bit wonky.  the first version we publish will be `4.0.0-beta`, and subsequent versions will be `4.0.0-beta-2`, etc., but npm considers those distinct. this command should work by specifying a package range explicitly starting on a beta release: `npm install fauna-shell@">=4.0.0-beta"`